deploy.sh

```diff
@@ -47,3 +47,13 @@ else
     echo "Publishing failed."
     exit 2
 fi
+# Pylint code quality check
+find . -name "*.py" -depth -exec pylint {} \; |   # Finding files with .py extension
+grep -oE "\-?[0-9]+\.[0-9]+" |                    # Extract the score from each message
+awk 'NR==1 || NR % 4 == 0' |                      # Previuos command will generate garbage numbers as weel, this cuts away unnecasary data
```

Comment:
Please make the comment in this line free from spelling errors.

Reply:
No need for these comments besides the one on line 50.